treeherder/webapp/api/utils.py

```diff
@@ -4,12 +4,14 @@
 
 import django_filters
 
-from treeherder.model.models import Repository
-
+# queries are faster when filtering a range by id rather than name
+# trunk: mozilla-central, mozilla-inbound, autoland
+# firefox-releases: mozilla-beta, mozilla-release
+# comm-releases: comm-beta
 REPO_GROUPS = {
-    'trunk': ['mozilla-central', 'mozilla-inbound', 'autoland'],
-    'firefox-releases': ['mozilla-beta', 'mozilla-release'],
-    'comm-releases': ['comm-beta', 'comm-release'],
+    'trunk': [1, 2, 77],
+    'firefox-releases': [6, 7],
+    'comm-releases': [38],
 }
```

Comment:
I couldn't find 'comm-release' in the repository table. Was `comm-releases` supposed to include comm-central or some other comm branch instead? If not, I think it makes sense to delete this line.

Reply:
It's quite surprising that passing IDs directly makes such a difference.

Could you give an example SQL statement for both (use eg django-debug-toolbar to output) so we can see there isn't something else going on?

If we do want to pass IDs directly, then I don't think hardcoding them here is the right thing to do. We could fetch them from the DB as an initial query before passing that to the main query.

Re comm-releases, good spot. Strangely the repository does exist (https://hg.mozilla.org/releases/comm-release/), it's just not been added to Treeherder. Perhaps they just haven't realised that they could ask for it to be added?